src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -28,11 +29,29 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
       },
       {
         extend: 'excel',
+        exportOptions: {
+          format: {
+            body: (data, row, column, node) => {
+              return column === 3 ?
+              moment.duration(data).asHours().toFixed(4).slice(0, -1) :
+              data;
+            },
+          }
+        },
         text: 'Excel',
         filename: `time-entries-${formatDate(new Date(), 'MM_dd_yyyy-HH_mm', 'en')}`
       },
       {
         extend: 'csv',
+        exportOptions: {
+          format: {
+            body: (data, row, column, node) => {
```

Comment:
This function is repeated so I suggest to extract it as a named one and reuse it.

Reply:
Thanks for your review @faustocv. This change was applied in the last commit.